sonic_platform_base/module_base.py

```diff
@@ -17,6 +17,35 @@ class ModuleBase(device_base.DeviceBase):
     # Device type definition. Note, this is a constant.
     DEVICE_TYPE = "module"
 
+    # Possible card types for modular chassis
+    MODULE_TYPE_SUPERVISOR = "SUPERVISOR"
+    MODULE_TYPE_LINE    = "LINE-CARD"
+    MODULE_TYPE_FABRIC  = "FABRIC-CARD"
+
+    # Possible card status for modular chassis
+    #Module state is Empty if no module is inserted in the slot
```

Comment:
Please add a space after `#`, and do for all other comments like this below.

Reply:
Done.